saleor/product/tasks.py

```diff
@@ -33,35 +35,37 @@ def _update_variants_names(instance, saved_attributes):
 
 
 @app.task
-def update_variants_names(product_type_pk, saved_attributes_ids):
+def update_variants_names(product_type_pk: List[str], saved_attributes_ids: List[str]):
     instance = ProductType.objects.get(pk=product_type_pk)
     saved_attributes = Attribute.objects.filter(pk__in=saved_attributes_ids)
-    return _update_variants_names(instance, saved_attributes)
+    _update_variants_names(instance, saved_attributes)
 
 
 @app.task
-def update_product_minimal_variant_price_task(product_pk):
+def update_product_minimal_variant_price_task(product_pk: str):
```

Comment:
`str` or `int`?

Reply:
when getting pks I'm used to strings

Reply:
```
In [5]: type(Product.objects.all().first().id)
Out[5]: int
```

Reply:
Bummer.

Reply:
Yep, as @korycins noticed database IDs should be rather typed as `int`.